src/libslic3r/PrintObject.cpp

```diff
@@ -816,6 +956,9 @@ bool PrintObject::invalidate_state_by_config_options(
             || opt_key == "max_bridge_length"
             || opt_key == "support_interface_top_layers"
             || opt_key == "support_critical_regions_only"
+            //SS: PolyHoles
```

Comment:
did we miss `hole_to_polyhole_twisted` here?

Reply:
The superslicer source is missing it there, but that doesn't mean it is correct. I will do the change, test it and see if there are any noticable regression.

Reply:
Yeah, we need to handle hole_to_polyhole_twisted here, otherwise toggle/untoggle this option won't invalid the slicing status hence users can't reslice it with the new changes.

Reply:
Ah that makes sense, very new to how slicers work, thankyou for explaining.